common/changes/@rushstack/heft-api-extractor-plugin/heft-1.0_2025-09-12-22-18.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Relase version 1.0.0",
```

```suggestion
      "comment": "Release version 1.0.0",
```

I think the others need to be updated as well